### ISSUE TITLE
Mouse tracking in Firefox: `<pre>` `<code>` elements with multiple lines of code do not report text under the mouse properly.

### ISSUE THREAD

**Issue by hwf1324** [nvaccess/nvda#17236]
### Steps to reproduce:

1. Open: https://www.nvaccess.org/files/nvda/documentation/developerGuide.html
2. Mouse over a line of text in a code block

### Actual behavior:

For most lines of text, mouseover text does not report text.
Text is reported only when the mouse moves over a small area of the left border.

### Expected behavior:

Report text on mouseover like in Chrome.

#### NVDA installed/portable/running from source:
install
#### NVDA version:
alpha-34198,67f6cb99（2025.1.0.34198）
#### Windows version:
Windows 11 Version 22H2 (OS Build 22621.4037)
#### Name and version of other software in use when reproducing the issue:
Firefox 131.0 (64 bit)
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by hwf1324** [comment]
CC: @jcsteh

**Comment by Adriani90** [comment]
Does this work in Chromium?

**Comment by hwf1324** [comment]
> Does this work in Chromium?

Yes, as I said in the issue description, this works fine in Chrome.

**Comment by jcsteh** [comment]
https://bugzilla.mozilla.org/show_bug.cgi?id=1942887

**Comment by hwf1324** [comment]
> https://bugzilla.mozilla.org/show_bug.cgi?id=1942887

I'm a little confused, is this the same issue?

**Comment by jcsteh** [comment]
Ah, I misinterpreted this slightly, sorry. Looking more closely, it might be the same underlying problem, but it also might not be. The outer `<pre>` is a block element, not an inline element, so it doesn't quite fit the same criteria.

**Comment by jcsteh** [comment]
Distilled test case:
`data:text/html,<pre><code>ab cd%0aef gh`
The "b cd" text isn't properly hit testable. This does look like the same bug, but the `<pre>` is a block element, which is different and puzzling.

**Comment by jcsteh** [comment]
https://bugzilla.mozilla.org/show_bug.cgi?id=1945036

**Comment by jcsteh** [comment]
Fixed in Firefox Nightly.
